app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
+    mask_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_to_buy += 1
+    if mask_to_buy > 0:
```

Comment:
_> 0_ is redundant here as well, _if mask_to_buy > 0:_ same as _if mask_to_buy:_